src/app/client/src/app/modules/program-dashboard/components/program-datasets/program-datasets.component.ts

```diff
@@ -734,7 +803,7 @@ export class DatasetsComponent implements OnInit, OnDestroy {
       this.filter = [];
       setTimeout(() => {
         this.isProcessed = false;
-      }, 5000);
+      }, 10000);
```

Comment:
why do you need a 10 sec delay?

Reply:
earlier the user was seeing a particular message which depends on this flag for just 5 secs although message required some 6-7 seconds normally to read so had to update this to 10 seconds